src/routes/JobsRoute.js

```diff
@@ -1,11 +1,10 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { get } from 'lodash';
 import { getSASParams } from '@folio/stripes-erm-components';
-import { FormattedMessage } from 'react-intl';
 import { StripesConnectedSource } from '@folio/stripes/smart-components';
 import { stripesConnect } from '@folio/stripes/core';
 import { Callout } from '@folio/stripes/components';
+import calloutLogic from './components/calloutLogic';
```

Comment:
Also can you change `calloutLogic` to maybe `renderCallout` or some other

Reply:
Renamed to `renderJobChangeCallout`, to allow for future callouts such as edit to be added